applications/GeoMechanicsApplication/tests/test_line_interface_elements.py

```diff
@@ -81,6 +82,71 @@ def assert_outputs_for_3_plus_3_line_interface_element(self, simulation):
             self.assertAlmostEqual(relative_displacement_vertical_element[index][1], expected_vertical_displacement)
 
 
+    def assertVectorsAlmostEqual(self, actual_vectors, expected_vectors):
+        for actual_vector, expected_vector in zip(actual_vectors, expected_vectors):
+            self.assertVectorAlmostEqual(actual_vector, expected_vector)
+
+
+    def test_multi_stage_3_plus_3_line_interface_element_with_dirichlet_conditions(self):
```

Comment:
Since these tests are also such that the prescribed displacements result in the same tractions as Neumann (and the other way around), we might be able to reduce duplication a bit (similar to the single-stage tests), since all the asserts can be the same on both tests

Reply:
Good catch! I have extracted a method that asserts the results of a multi-stage test.